src/db.c

```diff
@@ -501,6 +509,18 @@ int dbGenericDelete(serverDb *db, robj *key, int async, int flags) {
     return dbGenericDeleteWithDictIndex(db, key, async, flags, dict_index);
 }
 
+/* Add a volatile key for a hashtable with volatile fields */
```

Comment:
I think we should also document that this function will calculate the index, and suggest to use an alternative when the index is known.
```suggestion
/* Add a key with volatile items to the global tracking kvstore.  */
```